_posts/2023-06-29-run-and-debug-rust-verification-harnesses-natively-on-vscode.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title:  "Run and debug Rust verification harnesses from VS Code"
+---
+
+Kani is a verification tool that can help you systematically test properties about your Rust code.
+To learn more about Kani, check out the [Kani tutorial](https://model-checking.github.io/kani/kani-tutorial.html) and our [previous blog posts](https://model-checking.github.io/kani-verifier-blog/).
+
+## Introducing the Kani VS Code Extension
+
+We are delighted to introduce the [Kani VS Code Extension](https://marketplace.visualstudio.com/items?itemName=model-checking.kani-vscode-extension), which is now available on the VS Code marketplace.
```

Comment:
I'd slot in one tiny "summary sentence" after this, something like. "The extension integrates the Kani verifier into the native VS Code UI" I think that would improve the flow.